Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -919,7 +919,12 @@ - (void)_updateScrollViewForTransaction:(const WebKit::RemoteLayerTreeTransactio
     }
 
     bool scrollingEnabled = _page->scrollingCoordinatorProxy()->hasScrollableOrZoomedMainFrame() || hasDockedInputView || isZoomed || scrollingNeededToRevealUI;
-    [_scrollView _setScrollEnabledInternal:scrollingEnabled];
+
+    // FIXME: <rdar://99001670> Get the default list of allowed touch types from UIKit instead of
+    // hardcoding it (based on <https://developer.apple.com/documentation/uikit/uigesturerecognizer/1624223-allowedtouchtypes>).
```

Comment:
Comment should be updated to reflect the fact that we are no longer "hardcoding" the value.

(It would still be nice to get the list from UIKit, so we don't need to store it ourselves)